cartography/intel/aws/ecr.py

```diff
@@ -61,46 +61,59 @@ def load_ecr_repositories(neo4j_session, data, region, current_aws_account_id, a
 
 
 @timeit
-def load_ecr_repository_images(neo4j_session, repo_data, region, aws_update_tag):
+def transform_ecr_repository_images(repo_data):
+    """
+    Ensure that we only load ECRImage nodes to the graph if they have a defined imageDigest field.
```

Comment:
Do you know why/in which cases images don't have an imageDigest field?

I think it would be useful to look at an example and see what's going on there. This may be happening if the image is a "multiarch" image with a manifest list and we would have to see how to handle it.

Reply:
Good point, I'll file a ticket on this. I find it odd that we only encountered this issue recently; the existing cartography.intel.aws.ecr.load_ecr_repository_images() code would have run into this issue as well, see https://github.com/lyft/cartography/blob/6ab67f8895995f59bf2d0638275e69110d4598d4/cartography/intel/aws/ecr.py#L96